wrangler-proto/src/main/java/io/cdap/wrangler/proto/workspace/v2/WorkspaceDetail.java

```diff
@@ -21,17 +21,27 @@
 
 import java.util.List;
 import java.util.Objects;
+import javax.annotation.Nullable;
 
 /**
  * Full information about the workspace, including sample data
  */
 public class WorkspaceDetail {
   private final Workspace workspace;
   private final List<Row> sample;
+  @Nullable
+  private final byte[] sampleAsBytes;
```

Comment:
better to not have this `Nullable`, you can move the logic to convert sample to byte here so the byte[] always contain same thing as sample - https://github.com/data-integrations/wrangler/blob/develop/wrangler-storage/src/main/java/io/cdap/wrangler/store/workspace/WorkspaceStore.java#L217-L222

Reply:
Done. Reusing ObjectSerDe to convert instead of adding new methods.